server/src/connectionHandlers/onDefinition.ts

```diff
@@ -104,6 +104,28 @@ export function onDefinitionHandler (textDocumentPositionParams: TextDocumentPos
       })
       return definitions
     }
+    // Overrides
+    if (analyzer.isOverride(documentUri, position.line, position.character)) {
+      const overrideFile = bitBakeProjectScannerClient.bitbakeScanResult._confFiles.find((confFile) => {
+        return confFile.name === word
+      })
+
+      if (overrideFile?.path !== undefined) {
+        definitions.push(createDefinitionLocationForPathInfo(overrideFile.path))
+        return definitions
+      }
+
+      if (lastScanResult !== undefined) {
+        const targetPath = lastScanResult.includeHistory.find((includePath) => {
+          return includePath.ext === '.conf' && includePath.name === word
+        })
+
+        if (targetPath !== undefined) {
+          definitions.push(createDefinitionLocationForPathInfo(targetPath))
+          return definitions
+        }
+      }
+    }
```

Comment:
The feature doesn't do anything for me in .inc files like busybox.inc.
The override operator is also not properly hilighted in dark blue (it's in light blue)
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/08057194-40eb-44fb-90d8-020ab88f248f)

Reply:
tree-sitter returned multiple errors on the original itself. The first error came from the line `113`. I don't see anything wrong with it since the highlighting seems fine. I will open a ticket for this.

Reply:
Ticket: 14146. Put low priority for now since this is a tree-sitter issue. Will work on the fix in their repo later.

Reply:
Yes it works well in other .inc like wget.inc.